designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/UsageStatisticsReportsSettingsDeterminer.scala

```diff
@@ -122,13 +136,22 @@ class UsageStatisticsReportsSettingsDeterminer(
     }
   }
 
-  private[statistics] def determineQueryParams(): Future[Map[String, String]] =
```

Comment:
I guess we can remove: `[statistics]` now

Reply:
It is used in tests, after moving in `ScenarioStatisticsTest`